pkg/volume/util/atomic_writer.go

```diff
@@ -146,30 +146,30 @@ func (w *AtomicWriter) Write(payload map[string]FileProjection, setPerms func(su
 
 	// (2)
 	dataDirPath := filepath.Join(w.targetDir, dataDirName)
-	oldTsDir, err := os.Readlink(dataDirPath)
+	oldTSDir, err := os.Readlink(dataDirPath)
```

Comment:
I would say technically this belongs in its own commit or PR

Reply:
Yes, you are right.
Because the lint check reported an error, I made modifications here.
I will resubmit a PR about this. Thanks very much @haircommander

Reply:
@bzsuni please address the comment above and I'll lgtm this PR.

Reply:
If the spelling is not changed, CI check will report an error. 
Should I change the spelling in this PR, or in a separate PR?
@bart0sh

Reply:
Let's move everything unrelated to sets into a separate commit and I think this should be enough.

Reply:
I have moved the spelling check to #120876
Could you help me lgtm @bart0sh